### ISSUE TITLE
What are you planning to do next?

### ISSUE THREAD

**Issue by Simon-Meng** [MichaelLF107/ethbox#1]

**Comment by MichaelLF107** [comment]
I tried to run a pre-compiled OpenCL miner to execute with the UWP application, but this only works on PC, on the Xbox it just closes the application before the miner has time to load.

Now I'm trying to rewrite this miner: https://github.com/Ethereum-x/ethminer in C# but since I'm still on my second semester on my Computer Science class it will take a while.

If you want to start process feel free to contribute to this project.

TLDR: I thought it was going to be easy, but it isn't.
